assets/js/app.js

```diff
@@ -268,6 +268,25 @@ Hooks.ConversationList = {
         this.el.prepend(msg)
       }
     });
+
+    this.handleEvent("only_show", ({
+      ids
+    }) => {
+      Array.from(this.el.children).forEach((item) => {
+        if (ids.includes(parseInt(item.dataset.riderId))) {
+          item.classList.remove("hidden");
+        } else {
+          item.classList.add("hidden");
+        }
+      });
+    });
+
+    this.handleEvent("clear_search", ({}) => {
```

Comment:
```js
    this.handleEvent("conversation_list:clear_search", ({}) => {
```
Thoughts on namespacing events? I'm not sure that we'll run into clashing events, but couldn't hurt (unless the events are somehow scoped to the hook object, I don't know?)

Reply:
Oh yeah that's a great idea!